routing/router.go

```diff
@@ -696,6 +696,21 @@ func (r *ChannelRouter) FindBlindedPaths(destination route.Vertex,
 		// Don't bother adding a route if its success probability less
 		// minimum that can be assigned to any single pair.
 		if totalRouteProbability <= DefaultMinRouteProbability {
+			route := routeWithProbability.route.Hops
+			channelIDs := ""
+			for i, hop := range route {
```

Comment:
maybe add a new string method to the route struct which only prints out the chained chanids and maybe using `strings.Builder` is more clean ?